### PR TITLE
onepass: update resources

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/onepass.rb [hunk 1]
```diff
@@ -14,47 +14,33 @@ class Onepass < Formula
...
-  depends_on "pcre" => :build
-
-  # Homebrew's swig breaks M2Crypto due to upstream's undermaintained status.
-  # https://github.com/swig/swig/issues/344
-  # https://github.com/martinpaljak/M2Crypto/issues/60
-  resource "swig304" do
-    url "https://downloads.sourceforge.net/project/swig/swig/swig-3.0.4/swig-3.0.4.tar.gz"
-    sha256 "410ffa80ef5535244b500933d70c1b65206333b546ca5a6c89373afb65413795"
+  resource "m2crypto" do
+    url "https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.22.6rc4.tar.gz"
```
Comment (DomT4) [inline Library/Formula/onepass.rb]: I'm a bit "eh" about including unreleased resources here, even if it fixes the other problem. Any idea how long until this becomes stable?

File: Library/Formula/onepass.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Onepass < Formula
   url "https://github.com/georgebrock/1pass/archive/0.2.1.tar.gz"
   sha256 "44efacfd88411e3405afcabb98c6bb03b15ca6e5a735fd561653379b880eb946"
   head "https://github.com/georgebrock/1pass.git"
-  revision 1
+  revision 2
```
Comment (DomT4) [inline Library/Formula/onepass.rb]: Why the revision need here?

Comment (nijikon) [inline Library/Formula/onepass.rb]: So we build against new resources?

Comment (mikemcquaid) [inline Library/Formula/onepass.rb]: I think it's fine to leave as-is.

File: Library/Formula/onepass.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/onepass.rb]: Agreed.

Comment (nijikon) [inline Library/Formula/onepass.rb]: As discussed in saltstack PR, this is a stable release :) https://pypi.python.org/pypi/M2Crypto

Comment (mikemcquaid) [inline Library/Formula/onepass.rb]: Weird but :+1: from me.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/8654991afd14c8fbda8e2c9d85edaaadf0cd557a. Thanks @nijikon!

**Pull request closed without merging**